CONTRIBUTING.md

```diff
@@ -199,3 +200,34 @@ You may want to use [Github search](https://help.github.com/articles/searching-i
 | Next Release    | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22status%3A+next-release%22&type=Issues)  | Status     | Will be available in the next release                                                                                                                                                                                                                                               |
 | Duplicate       | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22resolution%3A+duplicate%22&type=Issues) | Resolution | Duplicates an existing issue                                                                                                                                                                                                                                                        |
 | Wontfix         | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22resolution%3A+wontfix%22&type=Issues)   | Resolution | Will no longer be worked on                                                                                                                                                                                                                                                         |
+
+## Release process
+
+To publish a new version for all packages that have updated, follow these steps:
+
+1.  `git checkout master`
+2.  Run `lerna version --conventional-commits` and verify each package's version change.
+
+    -   If you'd like to do a dry run first, you can also use the `--no-push` flag to see the commit and tags that lerna will apply. You can also use `lerna changed` to see an output of what packages will be updated
+    -   The `--conventional-commits` flag should automatically determine what version each package should be updated to. If this doesn't seem accurate, you can maually choose each version by just running `lerna version` without the flag.
+
+    > Note, if package A requires package B and package B recieves a minor update, package A will also recieve a minor update via lerna.
```

Comment:
recieve -> receive

Reply:
"I before E except after C or when sounded like A as in neighbor or weigh."

oh yeah checks out. good catch.